probe-rs-tools/src/bin/probe-rs/cmd/snapshots/probe_rs__cmd__chip__single_chip_output.snap

```diff
@@ -1,12 +1,11 @@
 ---
-source: probe-rs/src/bin/probe-rs/cmd/chip.rs
+source: probe-rs-tools/src/bin/probe-rs/cmd/chip.rs
 expression: output
 ---
 nrf52840_xxaa
 Cores (1):
     - main (Armv7em)
-RAM: 0x20000000..0x20040000 (256.00 KiB)
-RAM: 0x00800000..0x00840000 (256.00 KiB)
-NVM: 0x00000000..0x00100000 (1.00 MiB)
-NVM: 0x10001000..0x10002000 (4.00 KiB)
-
+RAM: 0x20000000..0x20040000 (256.0 kiB)
```

Comment:
This change is a side-effect of the byte-unit -> bytesize change. I'm personally fine with one digit of precision here, especially looking at the difference in dependencies.